quic/s2n-quic-core/src/event.rs

```diff
@@ -74,6 +74,34 @@ impl<'a> IntoEvent<&'a str> for &'a str {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct Range<T: Copy> {
```

Comment:
I don't think we need to newtype Range. The purpose of newtyping in the events is to make sure things are stable, even if we change the internal implementation. If there is a stable interface provided by core, then we should use it.